app/models/solr_document.rb

```diff
@@ -17,4 +17,8 @@ class SolrDocument
   # and Blacklight::Document::SemanticFields#to_semantic_values
   # Recommendation: Use field names from Dublin Core
   use_extension(Blacklight::Document::DublinCore)
+
+  def campus
+    first('campus_unit_ssm')
```

Comment:
I doubt there is, but just to double check, there aren't any records that have multiple campuses, right?

Reply:
no multiple campuses